RocketChat/Rocket.Chat.ReactNative
# fix: Listen focus and request members list

## Proposed changes

When the screen is focused again we fetch the members list again, so the list will keep updated after adding a new member.

## Issue(s)	

Closes #4644 

## How to test or reproduce

1. Click on a channel
2. Click on the channel name to go to the actions screen
3. On the actions screen click on members
4. Filter members to show all members
5. Click add users
6. Select a user and click next. the user should be populated on the members screen immediately

## Screenshots

## Types of changes




## Checklist



## Further comments
